scripts/pylib/twister/twisterlib/testinstance.py

```diff
@@ -73,6 +74,22 @@ def __init__(self, testsuite, platform, outdir):
         self.filters = []
         self.filter_type = None
 
+    def record(self, recording):
+        if recording:
+            if self.recording is None:
+                self.recording = recording.copy()
+            else:
+                self.recording.extend(recording)
+
+            filename = os.path.join(self.build_dir, "recording.csv")
+            with open(filename, "at") as csvfile:
```

Comment:
why not just use 'w' mode, if I run the same case multiply times, the output will be confused

Reply:
ok